packages/flutter/test/widgets/page_view_test.dart

```diff
@@ -368,6 +368,70 @@ void main() {
     expect(tester.getTopLeft(find.text('Idaho')), const Offset(790.0, 0.0));
   });
 
+  testWidgets('Page snapping disable and reenable',
+      (WidgetTester tester) async {
+    final List<int> log = <int>[];
+
+    Widget build({bool pageSnapping}) {
```

Comment:
nit: spaces inside the braces

Reply:
Fixed.  This contradicts what "flutter format" does, is that intentional?